static/src/components/ChangePassword.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import * as actionCreators from '../actions/auth';
+import Paper from 'material-ui/Paper';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import {change_password} from '../utils/http_functions';
+import axios from 'axios';
+import { browserHistory } from 'react-router';
+
+function mapStateToProps(state) {
+    return {
+        isRegistering: state.auth.isRegistering,
+        registerStatusText: state.auth.registerStatusText,
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return bindActionCreators(actionCreators, dispatch);
+}
+const style = {
+    marginTop: 50,
+    paddingBottom: 40,
+    paddingTop: 25,
+    width: '100%',
+    textAlign: 'center',
+    display: 'inline-block',
+    color: "white",
+    backgroundColor: "rgba(255, 255, 255, 0.25)",
+    fontFamily: "AvenirNext-Medium"
```

Comment:
This font doesn't work for some computer, maybe change it to:
```suggestion
    fontFamily: "Avenir"
```